src/main/java/com/conveyal/datatools/manager/models/transform/AppendToFileTransformation.java

```diff
@@ -0,0 +1,76 @@
+
+package com.conveyal.datatools.manager.models.transform;
+
+import com.conveyal.datatools.common.status.MonitorableJob;
+import com.conveyal.datatools.manager.models.TableTransformResult;
+import com.conveyal.datatools.manager.models.TransformType;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+public class AppendToFileTransformation extends ZipTransformation {
+
+    public static AppendToFileTransformation create(String csvData, String table) {
+        AppendToFileTransformation transformation = new AppendToFileTransformation();
+        transformation.csvData = csvData;
```

Comment:
This might be out of scope. Should there be a check to comfirm that each row of new data has the correct number of columns matching the provided table?

Reply:
The problem is we don't know what table we're going to be modifying. If it comes from the database, we can determine it, but if it's coming from fetch we might not know at this point.